cli/main.cpp

```diff
@@ -18,9 +18,11 @@ int main([[maybe_unused]] const int argc, [[maybe_unused]] const char* argv[]) {
   engines += ", RubyEngine";
   fmt::print("Initializing the RubyEngine\n");
   // Load and construct the RUBY engine on demand using dynamic loading (e.g. dlopen)
-  openstudio::util::DynamicLibrary rubyEngineLib(openstudio::getCurrentModuleDir() / "librubyengine.so");
+  openstudio::util::DynamicLibrary rubyEngineLib(openstudio::getCurrentModuleDir() / getSharedModuleName("rubyengine"));
   const std::function<ScriptEngineFactoryType> rubyFactory = rubyEngineLib.load_symbol<ScriptEngineFactoryType>("makeScriptEngine");
   const std::unique_ptr<openstudio::ScriptEngine> rubyEngine(rubyFactory(argc, argv));
+  rubyEngine->registerType<openstudio::Measure*>("openstudio::Measure *");
```

Comment:
Mac runtime fix: need to registerType here instead of inside ScriptEngine's ctor